config/locales/curation_concerns.en.yml

```diff
@@ -101,17 +102,29 @@ en:
         property: keywords
       title:
         property: name
+
```

Comment:
Is this all new? I'm not saying I disagree with it or anything, but was it meant to be here?